lib/luahid.c

```diff
@@ -49,12 +49,12 @@ static int luahid_register(lua_State *L);
 
 static const luaL_Reg luahid_lib[] = {
 	{"register", luahid_register},
-	{NULL, NULL}
+	{NULL, NULL},
```

Comment:
please do any style change as a separate commit or a fixup in the original one.. don't mix up into new developments.. it makes quite harder to separate what needs to be reviewed..